src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoNumberGroupSizes.cs

```diff
@@ -16,7 +16,7 @@ public static IEnumerable<object[]> NumberGroupSizes_TestData()
 
             // TODO: when dotnet/corefx#2103 is addressed, we should also check fr-FR
             if (!PlatformDetection.IsUbuntu1510 && !PlatformDetection.IsUbuntu1604 && !PlatformDetection.IsUbuntu1610
-                && !PlatformDetection.IsWindows7 && !PlatformDetection.IsWindows8x && !PlatformDetection.IsFedora24)
+                && !PlatformDetection.IsWindows7 && !PlatformDetection.IsWindows8x && !PlatformDetection.IsFedora24 && !PlatformDetection.IsFedora25)
```

Comment:
please do the fix in the file:

https://github.com/dotnet/corefx/blob/193379e620ed2d1d1a2402c18bc62807e4820893/src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoData.cs#L14

so we don't have to keep adding more Fedora versions.